types/swarm/swarm.go

```diff
@@ -54,13 +54,27 @@ type RaftConfig struct {
 	SnapshotInterval           uint64 `json:",omitempty"`
 	KeepOldSnapshots           uint64 `json:",omitempty"`
 	LogEntriesForSlowFollowers uint64 `json:",omitempty"`
-	HeartbeatTick              uint32 `json:",omitempty"`
-	ElectionTick               uint32 `json:",omitempty"`
+
+	// ElectionTick is the number of Node.Tick invocations that must pass between
```

Comment:
nit: `Node.Tick` is an etcd/raft internal concept. Maybe `ElectionTick is the number of ticks that must pass...`?